guide/react/installation.mdx

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Installation
+
+### 1. Install the package
+
+Install `@fishjam-cloud/react-client` package:
+
+<Tabs groupId="package-managers">
+
+  <TabItem value="npm" label="NPM">
+
+    ```
+    npm install @fishjam-cloud/react-native-client
+    ```
+
+  </TabItem>
+
+  <TabItem value="yarn" label="YARN">
+
+    ```
+    yarn add @fishjam-cloud/react-client
+    ```
+
+  </TabItem>
+</Tabs>
+
+### 2. Setup Fishjam context
+
+Wrap your app in our `FishjamProvider` component.
+
+```tsx
+import { FishjamProvider } from "@fishjam-cloud/react-client";
+
+ReactDOM.createRoot(document.getElementById("root")!).render(
```

Comment:
Does `!` after `getElementById` is standard in React app?

Reply:
Yes, [this is a standard](https://github.com/vitejs/vite/blob/main/packages/create-vite/template-react-ts/src/main.tsx).
The "root" div is by default always present in the boilerplate `index.html`.